src/fragments/lib/auth/js/advanced.mdx

```diff
@@ -400,60 +364,6 @@ Auth.configure({
 })
 ```
 
-This feature is also integrated into `aws-amplify-react`:
```

Comment:
This section is removed because latest Authenticator does not officially support open id flows. Currently `Authenticator` [only supports](https://ui.docs.amplify.aws/react/components/authenticator#social-providers) `facebook`, `google`, `apple` and `Amazon` out of the box.